sofy_sofy/README.md

```diff
@@ -0,0 +1,43 @@
+# Sofy
+
+## Overview
+
+### What is Sofy 
+SOFY is a no-code test automation platform that offers end-to-end testing from the CI/CD pipeline on real devices, making the testing process faster, more efficient, and more accessible than ever before. With SOFY, clients can automate the entire testing process, from test case planning to environment setup, and from reports to maintenance, without the need to write a single line of code. SOFY's intuitive interface sets it apart from other test automation platforms, allowing clients to achieve their software development goals with confidence.
+
+### Sofy Datadog Dashboard
+Integrating SOFY with Datadog allows you to gain even deeper insights into your testing process by visualizing key metrics and trends in the Datadog dashboard. With the ability to quickly and easily view how your device metrics are trending for each test case, you can identify potential issues early on and optimize your testing process for maximum efficiency. The Datadog dashboard provides real-time visibility into your SOFY test results, enabling you to monitor and analyze performance over time and make data-driven decisions to improve overall software quality. With this integration, you can take your testing and monitoring capabilities to the next level, ensuring that your applications perform flawlessly every time.
```

Comment:
We try to avoid marketing material within the overview section for integrations. The overview section should include:

1. What does Sofy do in 1-2 sentences?
2. What will customers get from this integration?

Can you please update the overview to reflect the above? Thanks!